Sources/TestingMacros/Support/ConditionArgumentParsing.swift

```diff
@@ -477,6 +466,22 @@ private func _parseCondition(negating expr: ExprSyntax, isParenthetical: Bool, f
 ///
 /// - Returns: An instance of ``Condition`` describing `expr`.
 private func _parseCondition(from expr: ExprSyntax, for macro: some FreestandingMacroExpansionSyntax, in context: some MacroExpansionContext) -> Condition {
+  // Handle closures with a single expression in them (e.g. { $0.foo() })
```

Comment:
Moved this up so that closure "expansion" (not really but sure why not) still happens. If we ever actually start doing fun things with closures, we'll probably need to move this back down?